python/lsst/rucio/register/dataset_map.py

```diff
@@ -1,4 +1,4 @@
-# This file is part of dm_rucio_register
+# This file is part of rucio_register
```

Comment:
Not related, but I believe `typing.Dict` has been deprecated since Python 3.9.

Reply:
Interestingly enough, that `ruff` command only fixed a couple of lines in a test file to change  `with open(jsonfile, "r") as f` to `with open(jsonfile) as f` - I'll fix the `typing.Dict` item above.